src/PathProcessor/OgSubdomainPathProcessor.php

```diff
@@ -7,73 +7,76 @@
 use Drupal\Core\PathProcessor\OutboundPathProcessorInterface;
 use Drupal\Core\Render\BubbleableMetadata;
 use Drupal\Core\Site\Settings;
+
 use Symfony\Component\HttpFoundation\Request;
 
 /**
- * Processes the inbound path using path alias lookups.
+ * Processes the path using by using subdomains attached to organic groups.
  */
-class OgSubdomainPathProcessor implements OutboundPathProcessorInterface, InboundPathProcessorInterface {
+class OgSubdomainPathProcessor implements InboundPathProcessorInterface, OutboundPathProcessorInterface {
 
-  /**
-   * To list all entity types.
-   * @var EntityTypeManagerInterface
-   */
   protected $entityTypeManager;
 
-  /**
-   * To get the base url from settings.php
-   * @var Settings
-   */
   protected $settings;
 
-  /**
-   * Constructs a PathProcessorAlias object.
-   *
-   * @param \Drupal\Core\Path\AliasManagerInterface $alias_manager
-   *   An alias manager for looking up the system path.
-   *
-   * @param \Drupal\Core\Entity\EntityStorageInterface $entity_storage
-   *    Entity storage for looking up the subdomain records on particular entities.
-   */
   public function __construct(Settings $settings, EntityTypeManagerInterface $entity_type_manager) {
     $this->settings = $settings;
     $this->entityTypeManager = $entity_type_manager;
   }
 
+  private function isPathSubbable($path) {
+      $invalid_starting_paths = array(
+          '/system',
+          '/admin',
+          '/node',
```

Comment:
'/node' is kind of dirty because out of the box, drupal in its simplest form can have a  node with a path like https://success.economist.com/node/2 

It would be a little more complicated to do: '/node/add', '/node/%/edit'.  

This falls in the bucket of things we (unl) don't need but if we wanted to generalize this module for community release, would be an issue